lib/Settings/MappingProfiles/listTemplate.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  FormattedMessage,
+  FormattedDate,
+} from 'react-intl';
+import {
+  get,
+  isEmpty,
+} from 'lodash';
+
+import { FOLIO_RECORD_TYPES } from '../../utils';
+
+export const listTemplate = {
+  name: record => record.name,
+  folioRecord: record => {
+    const { existingRecordType } = record;
+
+    return <FormattedMessage id={FOLIO_RECORD_TYPES[existingRecordType].captionId} />;
+  },
+  tags: record => {
+    const tags = get(record, 'tags.tagList', []);
+
+    if (isEmpty(tags)) {
+      return '-';
```

Comment:
Please use [NoValue](https://github.com/folio-org/stripes-components/blob/master/lib/NoValue/readme.md) for this.